site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,413 @@
+
+# Contents
+
+1. [New Features]({{< relref "#new-features" >}})
+1. [Android]({{< relref "#android" >}})
+1. [Emulation Accuracy]({{< relref "#emulation-accuracy" >}})
+1. [Frontend Improvements]({{< relref "#frontend-improvements" >}})
+1. [Surprise Announcement]({{<relref "#surprise-announcement" >}})
+1. [Conclusion]({{<relref "#conclusion" >}})
+
+# New Features
+
+## Implement basic rerecording features ([#5448](https://github.com/citra-emu/citra/pull/5448)) by [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+Following on from the implementation of a built-in video dumper, mentioned in our previous Progress Report, is the addition of more features to aid all of your video recording and TAS needs!
+
+Being able to use rerecording in an emulator is key when wanting to create TAS videos! These would include features such as adjusting the emulation speed and being able to sync-robust save states. Sync-robustness is the concept that if a save state is launched a certain number of times, it should behave the exact same way each time.
+
+This PR implements many basic rerecording features, following the [TASVideos](https://tasvideos.org/) requirements and desired features for a rerecording emulator, but also adds other needed features for our built-in video dumper such as separate save state slots for each movie, adding a read-only mode, fixing desync and file corruption bugs, and also remaking the UI to be squeaky clean!
+
+To utilize the rerecording features, head to `Tools -> Movie` to record your game as a .CTM (Citra TAS Movie) file. These .CTM files can then be played back from within Citra to use for TAS runs.
+
+{{< figure src="ctm.png"
+    title="" >}}
+
+## Implement Reverse Interlaced 3D ([#5580](https://github.com/citra-emu/citra/pull/5580)) by [oneup03](https://github.com/oneup03)
+
+Interlaced 3D is the way of displaying stereoscopic 3D content on passive 3D supported monitors and TV screens. This was [first implemented](https://github.com/citra-emu/citra/pull/5018) in Citra in 2019 in a bid to create more ways to view 3D on more devices. However, some monitors render each eye in a reverse order from normal interlaced monitors, such as LG OLED 3DTV's.
+This PR implements a new way to view the 3D capabilities on a new type of screen, keeping the 3D novelty of the 3DS that so many of us love alive.
+
+This setting can be found in `Emulation -> Configure` (`Citra -> Preferences` on MacOS), inside the `Graphics` tab. Make sure to set the **Depth** to 100% as well to utilize this feature.
+
+{{< figure src="rinterlaced.png"
+    title="" >}}
+
+## Custom folders for SDMC and NAND Directories ([#5759](https://github.com/citra-emu/citra/pull/5759), [#6014](https://github.com/citra-emu/citra/pull/6014), [#6157](https://github.com/citra-emu/citra/pull/6157)) by [nieldm](https://github.com/nieldm) and [SachinVin](https://github.com/SachinVin) and [vitor-k](https://github.com/vitor-k)
```

Comment:
nit: Why two ands? comma?

Reply:
Copied from previous progress report lol. A lot of formatting stuff is in here